### PR TITLE
Bump node to the latest LTS v16.13.0

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR bumps node to `v16.13.0` for our rest modules.

- Bump node to v16.13.0
- Switch to package lockfile v2
- Update test snapshot due to error message change

**Related issue(s)**:

**Notes for reviewer**:

Will sync with OPS to ensure they bump npm to `v8.1.0` as well.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rest/pom.xml [hunk 1]
```diff
@@ -38,8 +38,7 @@
                             <goal>install-node-and-npm</goal>
                         </goals>
                         <configuration>
-                            <nodeVersion>v14.16.0</nodeVersion>
-                            <npmVersion>6.14.11</npmVersion>
+                            <nodeVersion>v16.13.0</nodeVersion>
```
Comment (xin-hedera) [inline hedera-mirror-rest/pom.xml:41]: the plugin will install the npm version bundled with node, if the npm version is not explicitly specified.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Please update the versions in the top-level README.md and in `docs/operations.md`. Should also bump the NVM version and mirror node version mentioned in the latter as well.

File: docs/operations.md [hunk 2]
```diff
@@ -251,7 +251,7 @@ also recommended to create a `restapi` user and execute all commands as that use
 sudo apt-get install build-essential libssl-dev git postgresql-client
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
 bash # reload shell
-nvm install stable #pin the current versions node: v12.10.0, npm: 6.10.3
+nvm install --lts #pin the current lts version node: v16.13.0, npm: 8.1.0
```
Comment (steven-sheehy) [inline docs/operations.md:254]: Let's drop the versions in the comments so we don't have to maintain it.

File: docs/operations.md [hunk 3]
```diff
@@ -251,7 +251,7 @@ also recommended to create a `restapi` user and execute all commands as that use
 sudo apt-get install build-essential libssl-dev git postgresql-client
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
```
Comment (steven-sheehy) [inline docs/operations.md:252]:
```suggestion
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
```

File: docs/operations.md [hunk 4]
```diff
@@ -264,7 +264,7 @@ hedera:
         host: dbhost
         password: mirror_api_pass
 EOF
-wget "https://github.com/hashgraph/hedera-mirror-node/releases/download/v0.8.1/hedera-mirror-rest-v0.8.1.tgz"
+wget "https://github.com/hashgraph/hedera-mirror-node/releases/download/v0.45.0/hedera-mirror-rest-v0.45.0.tgz"
```
Comment (steven-sheehy) [inline docs/operations.md:267]: Here and below. Then add a note about setting `VERSION` appropriately so we don't have to keep it up to date.

```suggestion
wget "https://github.com/hashgraph/hedera-mirror-node/releases/download/${VERSION}/hedera-mirror-rest-${VERSION}.tgz"
```

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: docs/operations.md (hunk 4 above)
Comment (Nana-EC) [inline docs/operations.md:267]: nit: Should we either add a "Replace the version below with the appropriate version:" before the shell script like the `Upgrading` section,  or maybe we should replace both links with mirror node versions with a `{latest_version}` placeholder

**Review by Nana-EC — APPROVED** [review]
LGTM, minor nit if you circle back

File: docs/operations.md (hunk 4 above)
Comment (xin-hedera) [reply docs/operations.md:267]: adding it

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**